app-drawer-layout/app-drawer-layout.html

```diff
@@ -170,16 +178,19 @@
 
       _responsiveChange: function(narrow) {
         this._setNarrow(narrow);
-        this._resetLayout();
+        this.resetLayout();
       },
 
-      _resetLayout: function() {
-        if (this.isAttached) {
+      resetLayout: function() {
+        this.debounce('_resetLayout', function() {
+          if (!this.isAttached) {
+            return;
+          }
           this.drawer.opened = !this.narrow;
           this.drawer.persistent = !this.narrow;
           this.$.contentContainer.style.marginLeft = (this.narrow || (this.drawer.position == 'right')) ? '' : this.drawer.getWidth() + 'px';
           this.$.contentContainer.style.marginRight = (this.narrow || (this.drawer.position == 'left')) ? '' : this.drawer.getWidth() + 'px';
-        }
+        }, 1);
```

Comment:
we could try to avoid lines that are more than 100 characters per line. e.g.
```js
this.$.contentContainer.style.marginRight = (this.narrow || (position == 'left')) ?  '' : 
  this.drawer.getWidth() + 'px';
```

Reference: https://github.com/airbnb/javascript#18.12